src/lottery/assembly/index.ts

```diff
@@ -7,48 +7,48 @@ type AccountId = string;
 @nearBindgen
 export class Contract {
 
-  private _owner: AccountId
```

Comment:
`_` for private variables isn't really used -- exception is when overriding get / set methods for private class property
e.g.
```ts
private _var: string;
get var(): string {
  return this._var;
}
```